### DISCUSSION TITLE
No interrupt for ISO Async IN EP when declared TUSB_ISO_EP_ATT_IMPLICIT_FB

### DISCUSSION THREAD

**Discussion by mastupristi** [hathach/tinyusb#2476] [General]
Hi,
I'm using MIMXRT1170-EVK. I'm trying to use an implicit feedback endpoint.

I have a breakpoint here https://github.com/hathach/tinyusb/blob/master/src/portable/chipidea/ci_hs/dcd_ci_hs.c#L666

I have an Isochronous Asynchronous IN EP. When EP is declared **TUSB_ISO_EP_ATT_DATA** the breakpoint is regularly triggered, whereas if I declare it **TUSB_ISO_EP_ATT_IMPLICIT_FB** the breakpoint is never triggered.

where am i going wrong? Is this perhaps a bug?

Attached are the descriptor declaration files, of the two .h consider only usb_descriptors_implFb.h.

[usb_desc.zip](https://github.com/hathach/tinyusb/files/14310624/usb_desc.zip)

**Comment by HiFiPhile** [comment]
Which host OS are you testing ?

I believe Windows's UAC2 driver doesn't support implicit feedback.

**Reply by mastupristi** [reply]
Yes, you are right
